documents/knowledge/elements.yaml

```diff
@@ -3610,13 +3615,15 @@
     - "[4] : 5"
     - "[5] : 7"
     - "[69] : 71"
+    - "['3 * x ** 2 + 493 * x - 2319'] : 270877"
 
 - element: "∆ṗ"
-  name: First Prime Before a Number
+  name: First Prime Before a Number / Factorise Expression
```

Comment:
Factorise Expression -> Factor Expression? "Factor" is enough afaik, and it's compatible with both British and American English :P

Reply:
Factorise is what we say in BrE. "Factor" sounds very American and very wrong to me. And since `factorise` is a superset of the letters of `factor`, this is better for searchability